src/SFA.DAS.ProviderFunding.Web.UnitTests/Controllers/HomeControllerTests.cs

```diff
@@ -12,17 +11,24 @@ namespace SFA.DAS.ProviderFunding.Web.Tests.Controllers
 {
```

Comment:
I know you haven't changed this but this namespace is wrong, worth fixing?

Reply:
Good spot - done.